CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -164,13 +171,11 @@ public bool TrySetProperty(long objectId, string name, object value, out string
         /// <summary>
         /// Analyse the object and generate metadata which will
         /// be used by the browser subprocess to interact with Cef.
-        /// Method is called recursively
+        /// Method is NOT called recursively and use late binding instead
```

Comment:
Just remove this line in the comment. Doesn't really make sense to say what it doesn't do.